src/utils.rs

```diff
@@ -81,9 +81,9 @@ pub fn snippet_block<'a>(cx: &Context, span: Span, default: &'a str) -> Cow<'a,
 /// Trim indentation from a multiline string
 /// with possibility of ignoring the first line
 pub fn trim_multiline(s: Cow<str>, ignore_first: bool) -> Cow<str> {
-    let s = trim_multiline_inner(s, ignore_first, ' ');
-    let s = trim_multiline_inner(s, ignore_first, '\t');
-    trim_multiline_inner(s, ignore_first, ' ')
+    let s_space = trim_multiline_inner(s, ignore_first, ' ');
```

Comment:
> Also s_space is not necessarily the same string as s. And by the way, it's a Cow<str>, not a string.

Sure, but conceptually it's a string that is threaded through different processing passes. Let's see what @Manishearth thinks.